lmnet/tests/lmnet_tests/networks_tests/test_quantized_network_arguments.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+
+from lmnet.networks.classification.lmnet_v0 import LmnetV0Quantize
+from lmnet.networks.classification.lmnet_v1 import LmnetV1Quantize
+from lmnet.networks.classification.darknet import DarknetQuantize
+from lmnet.networks.object_detection.lm_fyolo import LMFYoloQuantize
+from lmnet.networks.object_detection.yolo_v2_quantize import YoloV2Quantize
+from lmnet.networks.segmentation.lm_segnet_v0 import LmSegnetV0Quantize
+from lmnet.networks.segmentation.lm_segnet_v1 import LmSegnetV1Quantize
+from lmnet.quantizations import (
+    binary_mean_scaling_quantizer,
+    linear_mid_tread_half_quantizer,
+)
+
```

Comment:
I hope to implement unit tests using `QuantizeParamInit` mixin class with new network class 
* When the param `quantize_first_convolution=True` and `quantize_last_convolution=True`, the all of network's activations before convolution and weights are quantized.

* When the param `quantize_first_convolution=True` and `quantize_last_convolution=False`, the all of network's activations before convolution and weights are quantized excluding last convolution.

* When the param `quantize_first_convolution=False` and `quantize_last_convolution=True`, the all of network's activations before convolution and weights are quantized excluding first convolution.